v03_pipeline/lib/reference_datasets/config.py

```diff
@@ -0,0 +1,34 @@
+from v03_pipeline.lib.model import ReferenceGenome
+from v03_pipeline.lib.reference_data.cadd import load_cadd_ht_from_raw_dataset
+from v03_pipeline.lib.reference_data_OLD.hgmd import download_and_import_hgmd_vcf
+
+CONFIG = {
+    'cadd': {
+        'load_parsed_dataset_func': load_cadd_ht_from_raw_dataset,
+        ReferenceGenome.GRCh37: {
+            'version': '0.0',
```

Comment:
is there a reason we are starting at version 0 instead of version 1?

Reply:
no I have no strong opinion on this.. 1.0 sounds more trustworthy though so let's do that